app/Config/bootstrap.php

```diff
@@ -48,6 +48,18 @@
  *
  */
 
+/**
+ * Full URL prefix
+ * Get and set the fullBaseUrl using an environmental variable
+ * e.g. Apache server configuration
+ * SetEnv COMANAGE_REGISTRY_FULL_BASE_URL https://example.com
+ */
+if (!defined('CMG_FULL_BASE_URL')
```

Comment:
Do we really need to defined `CMG_FULL_BASE_URL`? If someone wants to define the URL via Cake configuration, they can just set `App.fullBaseUrl` directly (and not pass in `$COMANAGE_REGISTRY_FULL_BASE_URL`).